README.md

```diff
@@ -255,6 +256,424 @@ following the ASF Policy.
 
   ![Code](https://raw.githubusercontent.com/apache/airflow/main/docs/apache-airflow/img/code.png)
 
+## Database scheme
```

Comment:
Is it necessary to put this in README? I’m not sure that’s particularly useful TBH…

Reply:
Not really, we should find a better place to embed it. It was just the first markdown file I found. Any suggestions?

Reply:
The user-facing documentation already has an image, so maybe somewhere in `contributing-docs`? IIRC there’s a document there on database models.

Reply:
Unfortunately I can't inject the mermaid in the rst file so I just replace the svg file instead
and restore the original README
https://github.com/dennysreg/airflow/blob/issue-36842-generate-a-mermaid-erd-diagram-from-sqlalchemy-models/docs/apache-airflow/database-erd-ref.rst